scripts/pulse-results/README.md

```diff
@@ -0,0 +1 @@
+this directory contains the json output from `process_a11y` script
```

Comment:
And, please use the `utils.results_dir()` or `utils.cache_dir()` method (or make up a new method) that ensures that the results get dropped in the `--output` directory, if one is given. This is critical to production operation.